tutorials/provisional/mosaic.py

```diff
@@ -123,14 +123,30 @@ def identify_axes(ax_dict, fontsize=48):
 ax_dict = fig.subplot_mosaic(mosaic)
 identify_axes(ax_dict)
 
+###############################################################################
+# Alternatively, you can use the more compact string notation
+mosaic = "AB;CD"
+
+###############################################################################
+# will give you the same composition, where the ``";"`` is used
+# as the separator instead.
```

Comment:
```suggestion
# as the row separator instead.
```
?

Reply:
(Should one somehow indicate that `mosaic = "AB\nCD"` is the same thing in the previous example? To me that seems like a more common way to write it than the multi-line solution above.)

Reply:
I wouldn`t bother with `"AB\nCD"`. That's technically possible, but not reasonable. If you want to write it in one line, the `;` separator is more readable.